templates/views/movie_profile.pug

```diff
@@ -6,7 +6,15 @@ include ../mixins/social.pug
 
 block content
   
-  - const sections = [ "context", "synopsis", "cinematography", "pointsForDiscussion", "externalLinks", ]
+  //- Helper
+  - const firstKey = (obj) => Object.keys(obj)[0]
+  
+  //- Additional sections possible
+  - const sections = [ "context", "synopsis", "cinematography", "pointsForDiscussion", "externalLinks" ]
+  
+  //- Query checks
+  - const hasVideo = movie.video
```

Comment:
Can you set these values explicitly.

ie
```js
const hasVideo = movie.video !== undefined
const hasImages = (movie.gallery !== undefined) && movie.gallery.length !== 0
```

This is the same as using strict inequality.

Ie what is the case where movie.video = "" truthy or falsy, Reading you code is is not clear if video being an empty string is falsy because you want it to be, or that is a bug.

Short answer: Always explicitly set something to be true or false, dont let js coerce something into a bool.